manifests/plugin/openscap/params.pp

```diff
@@ -0,0 +1,7 @@
+# Default parameters for the Openscap smart proxy plugin
+class foreman_proxy::plugin::openscap::params {
+  $enabled                 = true
```

Comment:
I don't follow, why `enabled` should be undef? Also it does not seem to be true for same attribute in pulp plugin https://github.com/theforeman/puppet-foreman_proxy/pull/140/files#diff-fdc8c6b8036bbd5f1bb55daacf07098bR3